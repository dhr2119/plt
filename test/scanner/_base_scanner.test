( ) { } [ ]
if else elsif while for return main break
== != < > <= >= && || !
+ - * / =
int float bool void null String true false
; , :
hi 99
