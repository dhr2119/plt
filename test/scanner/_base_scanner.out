LPAREN
RPAREN
LBRACE
RBRACE
LBRACKET
RBRACKET
IF
ELSE
WHILE
FOR
RETURN
MAIN
BREAK
EQ
NEQ
LT
GT
LEQ
GEQ
AND
OR
NOT
PLUS
MINUS
TIMES
DIVIDE
ASSIGN
INC
DEC
INT
FLOAT
BOOL
VOID
NULL
STRING
TRUE
FALSE
SEMI
COMMA
COLON
ID
INT_LIT
STRING_LIT
FLOAT_LIT
